lib/blacklist.js

```diff
@@ -15,7 +15,7 @@ const getBlacklist = () => {
 // Delete row from blacklist table by crypto code and address
 const deleteFromBlacklist = (cryptoCode, address) => {
   const sql = `DELETE FROM blacklist WHERE crypto_code = $1 AND address = $2`
-  notifier.clearBlacklistNotification(cryptoCode, address)
+  notifierQueries.clearBlacklistNotification(cryptoCode, address).catch(console.error)
```

Comment:
Are we ever doing something with these errors other than printing to log? If not, ``notifierQueries`` should take care of it.